felix/calc/calc_graph_bench_test.go

```diff
@@ -104,6 +125,13 @@ func benchInitialSnap(b *testing.B, numEndpoints int, numLocalEndpoints int, num
 		b.ReportMetric(float64(time.Since(startTime).Seconds()), "s")
 		b.ReportMetric(float64(numMessages), "Msgs")
 	}
+	b.StopTimer()
+	runtime.GC()
+	time.Sleep(time.Second)
+	var m runtime.MemStats
+	runtime.ReadMemStats(&m)
+	b.ReportMetric(float64(m.HeapAlloc/(1024*1024)), "HeapAllocMB")
+	runtime.KeepAlive(cg)
```

Comment:
I think it would help to add some comments to these lines to explain your purpose.